docs/index.asciidoc

```diff
@@ -23,7 +23,14 @@ include::{include_path}/plugin_header.asciidoc[]
 
 Send events over UDP
 
-Keep in mind that UDP will lose messages.
+Keep in mind that UDP does not provide delivery or duplicate protection guarantees;
+even when this plugin succeeds at writing to the UDP socket, there is no guarantee that
```

Comment:
The extra description suggested by yaauie is a nice addition. I suggest breaking the first sentence into two sentences for easier reading and translation. 

"Keep in mind that UDP does not provide delivery or duplicate protection guarantees.
Even when this plugin succeeds at writing to the UDP socket..."

Reply:
I'm okay with this either way. I love semicolons, but a full stop works here too.

Reply:
I love semicolons, too. <3  For tech docs that will be translated into other languages or used "as is" by non-native speaker, simpler constructions are usually better.

Reply:
applied suggestiuon.